### DISCUSSION TITLE
Assigning a blueprint to a template

### DISCUSSION THREAD

**Discussion by MantasZakas** [statamic/cms#8171] [Troubleshooting]
I am pretty new to Statamic and while this might be obvious, I just can't the answer. Is there a way to bind a blueprint to a template? If I change the template of a page to say "About us" I would like to see all the fields that I need, without having to create a new page with the correct blueprint.

**Comment by jasonvarga** [comment]
If it's an existing page, there isn't really a way to do it in the control panel. You can change `blueprint: whatever` in the markdown file, though.
